src/is.js

```diff
@@ -142,4 +169,5 @@ export default {
     regexp: isRegexp,
     undefined: isUndefined,
     windowObject: isWindowObject,
+    all: matchesAll,
```

Comment:
`all` is not intended to be part of `is` checker. But rather a different checker, as described in the issue.